landscape/lib/uaclient.py

```diff
@@ -0,0 +1,25 @@
+from landscape.client import IS_CORE
+from landscape.client import IS_SNAP
+
+if not IS_SNAP and not IS_CORE:
+    from uaclient.status import status
+    from uaclient.config import UAConfig
```

Comment:
Although this seems like it makes it obvious that the function below will not work for the snap and core, I think it's still a good idea to make sure it doesn't completely crash if for some reason it were imported and called.

Maybe have `get_pro_status` emit a warning and return an empty response?